lib/screens/lista_transferencias.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:bytebank/components/progresso.dart';
+import 'package:bytebank/models/transferencia.dart';
+import 'package:bytebank/webclient/transferencia_webclient.dart';
+import 'package:flutter/material.dart';
+
+const _tituloAppBar = 'Histórico';
+
+class ListaTransferencias extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() {
+    return _ListaTransferenciasState();
+  }
+}
+
+class _ListaTransferenciasState extends State<ListaTransferencias> {
+  final List<Transferencia> _transferencias = List();
+
+  @override
+  void initState() {
+    super.initState();
+    buscaTodas();
+  }
+
+  void buscaTodas() async {
+    final transferenciasEncontradas = await TransferenciaWebClient().todas();
+    setState(() {
+      _transferencias.addAll(transferenciasEncontradas);
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: Text(_tituloAppBar),
+        ),
+        body: configuraWidget());
+  }
+
+  Widget configuraWidget() {
```

Comment:
Should this be private or not? (others are)

Reply:
yep